nyudlts/ultraviolet
# Update Documentation

This PR will do the following:

- Remove policy and users docs, solves INVENIO-152
- Update docs homepage, solves INVENIO-153
- Add invenio-cli cheatsheet
- Add project orientation content and updated links
- Update Ruby Version to 3.0.5, solves INVENIO-151
- Update gem github-pages
- Update homepage Links
- Update docker documents
- Update Look and Feel documentation
- solves INVENIO-150

To test this PR make sure to:
- install newer version of Ruby with `rbenv install`
- install bundler with `gem install bundler`
- reinstall gems with `bundler install --redownload`
- run the project locally with `bundle exec jekyll serve`

To test this PR please check Github Actions [`pages-build-deployment`](https://github.com/nyudlts/ultraviolet/actions/workflows/pages/pages-build-deployment) to ensure it can run on this pipeline. I'm assuming that the GH action will read the `.ruby-version` file, but have no direct way to prove that it will build accordingly.

Thank you for our review!
Please let me know if there are changes needed :)
